### DISCUSSION TITLE
Help sending to the Segwit BTC Address

### DISCUSSION THREAD

**Discussion by txtr99** [ctubio/Krypto-trading-bot#1141] [Q&A] [answered]
Hi there, I would like to support this project like many of you, but for some reason I cannot withdraw any BTC from my Binance Wallet to that address given, they say they don't support addresses starting with b1p. Even though it starts with a 3, from googling it seems this is a newer version of Segwit. 

So I thought ok I'll transfer to my TrustWallet and then transfer from there but nope I cannot withdraw to my trustwallet BTC address either!!

Since I'm in Canada and none of my Cards actually work to buy crypto on any portal I've tried, I use P2P Binance to get my crypto. So I'm stuck right now with funds there. 

Anyways just wondering if anyone experienced this and knows a way I can get around it? Binance must be backed up, they haven't answered in hours and I'm not sure if it's a temporary issue or one of those indefinite lack of functionality things. 

Thanks a lot in advance!

PS, I checked the "GitHub Sponsor" area but it was not obvious to me that we could unlock this software via the Sponsor feature yet.  Maybe I'm wrong..

**Comment by ctubio** [comment, marked as answer]
:koala: many thanks for the sponsor, just let me know the btc address of the bot that you wish to unlock and i will do it for you

(btw, i never had issues with transactions out of binance so i can't help you there :S)

**Reply by txtr99** [reply]
Hey Carles! Absolutely! This is an answer to prayers for me. I became fascinated with Market Making recently and tried to make one on another bot platform, but it was too slow (10 second cycle/update times, no websockets) so I tried making my own in Python but.. well I'm just starting to learn that language and it was ugly and buggy...

Finding this - I was blown away.  In fact, do you offer any Tutoring on the UI and setup? There are a couple things I can't seem to find in the docs or understand such as "Width Too High" messages on the Quote Columns etc.

As for the API Key, actually in the meantime I got stubborn because I realized I have to know how to get my funds out of Binance when they block withdrawals for BTC since I could also not even withdraw to my own Wallet...

So I found a way to withdraw BEP20 BNB and found a Site to exchange it to BTC and managed to already send that and it already unlocked my software.  So I suppose now I should back up the API Key or the licence file.. I'll look into that, in case of hard drive corruption since there is no email trail.

When I saw the IRC channel I remembered the days of mIRC scripting back 15 years ago, nostalgic hadn't thought about it for ages.

Yea, looking good!

**Reply by ctubio** [reply]
:koala: i see you joined the IRC channel; feel free to ask any quesiton there

(i answer faster on private messages than in the public channel) :woman_dancing:
